CHANGELOG.rst

```diff
@@ -2,6 +2,35 @@
 SDK Changelog
 =============
 
+3.3.0-alpha.0 (2025-08-?)
+==================
+
+First preview release of the upcoming 3.3.0 version.
+
+New features
+------------
+
+* Switched from local storage to session storage for form submissions.
+* [#850] Provided explicit exports for our formio module and template library.
+* [#825] Added childrens component.
+
+Bugfixes
+--------
+
+* Fixed the behaviour of `selected` property.
+
+Project maintenance
+-------------------
+
+* Fixed an issue reported by npm audit.
+* Added Storybook stories for ``FormStart``.
+* Removed obsoleted and unused SDK embed example.
+* [open-formulieren/open-forms#4879] remove of_payment_action usage.
```

Comment:
```suggestion
* [#4879] Removed some dead code.
```
it's a bit uncommon, but we just use the ticket IDs in changelogs, from the actual number you can usually tell which repo it belongs to (SDK is < 1000).